Sources/StreamChat/WebSocketClient/Events/MessageEvents.swift

```diff
@@ -59,19 +59,17 @@ public struct MessageDeletedEvent<ExtraData: ExtraDataTypes>: EventWithUserPaylo
     }
 }
 
-public struct MessageReadEvent<ExtraData: ExtraDataTypes>: EventWithUserPayload, EventWithChannelId {
+public struct MessageReadEvent: EventWithUserPayload, EventWithChannelId {
     public let userId: UserId
     public let cid: ChannelId
     public let readAt: Date
-    public let unreadCount: UnreadCount
     
     let payload: Any
     
-    init(from response: EventPayload<ExtraData>) throws {
+    init<ExtraData: ExtraDataTypes>(from response: EventPayload<ExtraData>) throws {
         userId = try response.value(at: \.user?.id)
         cid = try response.value(at: \.cid)
         readAt = try response.value(at: \.createdAt)
-        unreadCount = try response.value(at: \.unreadCount)
```

Comment:
I'm confused, backend doesn't send this info anymore, or it never sent this, or this is not useful?

Reply:
This was actually a complete misunderstanding of how `MessageReadEvent` work :) The unread count is always zero hence is not sent. This unread count is the total unread count and is sent only for the current user -> see the PR description for more info.

Reply:
Shouldn't it be optional and useful when available? If it exists, we can use it to update current user's unread count